packages/forma-36-react-components/src/components/Typography/Typography/Typography.tsx

```diff
@@ -3,36 +3,38 @@ import cn from 'classnames';
 
 const styles = require('./Typography.css');
 
-export interface TypographyProps {
+export type TypographyProps = {
   extraClassNames?: string;
   children?: React.ReactNode;
+  style?: React.CSSProperties;
   testId?: string;
-}
+} & typeof defaultProps;
+
+const defaultConfiguration = {
+  displayText: { spacing: 'l' },
+  displayTextLarge: { spacing: 'xl' },
+  heading: { spacing: 'm' },
+  paragraph: { spacing: 'm' },
+  sectionHeading: { spacing: 'l' },
+  subheading: { spacing: 'm' },
+};
+
+const defaultProps = {
+  testId: 'cf-ui-text-container',
+};
 
-export const TypographyContext = React.createContext({});
+export const TypographyContext = React.createContext(defaultConfiguration);
```

Comment:
@suevalov this doesn't work as expected - it'll mean the typography components will _always_ have spacing applied.

https://deploy-preview-129--forma-36-react-components.netlify.com/?path=/story/components-typography-displaytext--default

In this Story there should be no margins applied

Reply:
Good catch, @m10l! I pushed the fix.

Reply:
Awesome